src/main/webapp/scripts/auth.js

```diff
@@ -0,0 +1,156 @@
+/**
```

Comment:
Can you refactor to remove duplication between this and https://github.com/googleinterns/step191-2020/blob/master/src/main/webapp/scripts/main.js? Maybe just split out the counter-specific code into its own file and include main.js in create-game.html along with any other js files that deal with this view.

Reply:
Then I will need to remove the duplicated stuff from the `main.js `to another script specially for Firebase since `main.js` tries to modify elements that do not exist on `create-game.js`.

Reply:
Yes. I think you'll eventually have something like this

```html
<script src="scripts/main.js"></script>
<script src="scripts/create-game.js"></script>
```

main.js has anything that all pages need. create-game.js has code specific to creating games.

Reply:
Done